src/views/Topic.vue

```diff
@@ -478,6 +482,17 @@ export default {
         images: _.get(this.$store.state, [ 'images', this.uuid ])
       }
     },
+    needWineWarning () {
+      const wineTopicIds = [
+        '5c25f9e3315ec51000903a82',
+        '5d22bb9fe311f3925c49396c',
+        '5a4d8e60160ac91000294611'
+      ]
```

Comment:
move `wineTopicIds` to this vue file constant block (line 126)  and use UPPER CASE